source/code-snippets/usage-examples/changeStream.js

```diff
@@ -0,0 +1,31 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection
+// string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a Change Stream on the movies collection
+    const changeStream = await collection.watch();
+    // use the next() function on the changeStream iterator to access the next event
```

Comment:
Nitpick: I'm picking on my own comment about your comment, but we should probably use "method" instead of "function" to stay consistent.

Reply:
fair enough - fixed